### PR TITLE
Quick fix to address unicode issues with character \u2013

### PR DESCRIPTION
**Pull request by dr-rodriguez**
This is a very quick script to address the en dash character which affected several Source names and even one spectrum url.

### CODE REVIEW

File: data/2mass_j11074656-7615174.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
             "epoch": 2000.0,
             "equinox": "J2000",
             "shortname": null,
-            "reference": "Góme01",
+            "reference": "G\u00f3me01",
```
Comment (kelle) [inline data/2mass_j11074656-7615174.json:10]: Oops. This went the wrong way!

File: data/2mass_j11123099-7653342.json [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
         {
-            "source": "2MASS J11123099–7653342",
+            "source": "2MASS J11123099-7653342",
             "ra": 168.129,
             "dec": -76.8928,
             "epoch": 2000.0,
             "equinox": "J2000",
             "shortname": null,
-            "reference": "Lópe04",
+            "reference": "L\u00f3pe04",
```
Comment (kelle) [inline data/2mass_j11123099-7653342.json:10]: Ooops.

File: data/Publications.json [hunk 3]
```diff
@@ -6852,13 +6852,13 @@
         "description": "ISOCAM observations of the Chamaeleon I dark cloud"
     },
     {
-        "publication": "Lópe04",
+        "publication": "L\u00f3pe04",
```
Comment (kelle) [inline data/Publications.json:6855]: Ooops.

File: data/Publications.json [hunk 4]
```diff
@@ -6852,13 +6852,13 @@
...
         "bibcode": "2004A&A...416..555L",
         "doi": "10.1051/0004-6361:20031720",
         "description": "The brown dwarf population in the Chamaeleon I cloud"
     },
     {
-        "publication": "Góme01",
+        "publication": "G\u00f3me01",
```
Comment (kelle) [inline data/Publications.json:6861]: Ooops.

**Review by kelle — CHANGES_REQUESTED** [review]

File: data/2mass_j11074656-7615174.json (hunk 1 above)
Comment (dr-rodriguez) [reply data/2mass_j11074656-7615174.json:10]: Yea, it's weird but a consequence. It is fixed in the second PR, though.

**Comment by dr-rodriguez** [comment]
I'll withdraw this PR since the fixes for the references depend on PR #267 (and that PR has merged this one already)

**Pull request closed without merging**